plugin.service.gto/addon.xml

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.4" provider-name="Birger Jesch">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.service" library="starter.py" start="startup">
```

Comment:
The addon use some methods to create dynamic lists that's only available in xbmcplugin (addDirectoryItem, endOfDirectory) and make use of plugin handles to actualize those content on home screen. Its debatable whether the addon is a script or a plugin or whatever.

The service is really a service which updates the content of the widget periodically.